src/js/common/schemaform/validation.js

```diff
@@ -78,9 +78,9 @@ export function transformErrors(errors, uiSchema) {
  * should call addError to add the error.
  */
 
-export function uiSchemaValidate(errors, uiSchema, formData, otherData, path = '') {
+export function uiSchemaValidate(errors, uiSchema, formData, formContext, path = '') {
```

Comment:
Sort of. `otherData` was originally meant to be the form data other than the current page. `formContext` is an object that contains any custom data we need to pass to the components the schema form library renders. It would likely contain `otherData` at some point. 

We haven't needed this yet, so it may change again. I'm not sure of the exact use case yet.